pydicom/sopclass_uids.py

```diff
@@ -0,0 +1,299 @@
+"""
+Storage SOP Class UIDs auto-generated by generate_storage_sopclass_uids.py
+"""
```

Comment:
Not to nitpick here, but shouldn't the generated filename be `_storage_sopclass_uids.py` since it's just related to the Storage SOP Classes?